pkgs/by-name/naja/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  bison,
+  boost,
+  capnproto,
+  doxygen,
+  flex,
+  libdwarf-lite,
+  pkg-config,
+  python3,
+  tbb_2021_11,
+  zstd,
+}:
+stdenv.mkDerivation {
+  pname = "naja";
+  version = "0-unstable-2024-07-21";
+
+  src = fetchFromGitHub {
+    owner = "najaeda";
+    repo = "naja";
+    rev = "8c068f3bd1bbd57b851547f191a58a375fd35cda";
+    hash = "sha256-aUYPJGr4D5n92fp0namPT6I/gMRZoF7YHnB7GoRzwYI=";
+    fetchSubmodules = true;
+  };
+
+  outputs = [
+    "dev"
+    "lib"
+    "out"
+  ];
+
+  nativeBuildInputs = [cmake];
+
+  buildInputs = [
+    bison
+    boost
+    capnproto
+    doxygen
+    flex
+    libdwarf-lite
+    pkg-config
+    python3
+    tbb_2021_11
+    zstd
+  ];
```

This'll break with `strictDeps`… and when our CMake gets patched to treat `nativeBuildInputs` and `buildInputs` `PATH` separation better: https://github.com/NixOS/nixpkgs/pull/318226

```suggestion
  nativeBuildInputs = [
    bison
    capnproto
    cmake
    doxygen
    flex
    pkg-config
    (python3.withPackages (ps: with ps; [
      sphinx
    ])) 
  ];

  buildInputs = [
    boost
    flex # include dir
    libdwarf-lite
    openssl # capnproto
    tbb_2021_11
    zlib # capnproto
    zstd
  ];
```

Only unsure about if `capnproto` should also be in `buildInputs` for its CMake modules.